tdr_snapshot_examples/get_snapshot_with_drs_and_download.py

```diff
@@ -0,0 +1,62 @@
+# This file goes through the full flow of getting a snapshot,
+# getting it's report's drs info, and then downloading the report
+# to the file system.
+
+import json
+
+from get_file_info_by_drs_path import get_file_info_by_drs_path
+from get_snapshot import get_snapshot
+from common.obtain_token import obtain_session
+
+
+def download_file_from_drs(session, drs_json, headers):
+    """
+    This function downloads file information by using the link inside drs_json
+    and saves it to your directory. This includes both pdf files and json files.
+
+    Args:
+        session: The session in which the order will be created. If no session is given, then
+                one will be obtained in the function.
+        drs_json: The json formatted file information obatined from the drs path of the file.
+        headers: The headers of the get request.
+    """
+    resp = session.get(url=drs_json["accessUrl"]["url"], headers=headers)
+    with open(f"{drs_json['fileName']}", "wb") as report_file:
+        report_file.write(resp.content)
+        report_file.close()
+
+
+if __name__ == "__main__":
+    session = obtain_session()
+
+    headers = {
+        "User-Agent": "GPO Bound Query",
+        "Content-Type": "application/json",
+    }
```

Comment:
I think this should probably go directly in the method instead of being passed in. Sorry, my bad.